### PR TITLE
Feat/some improvements

### PR DESCRIPTION
**Pull request by Antonboom**
Closes
- https://github.com/kazhuravlev/options-gen/issues/10
- https://github.com/kazhuravlev/options-gen/issues/8
- https://github.com/kazhuravlev/options-gen/issues/7

### CODE REVIEW

File: internal/generator/templates/options.go.tpl [hunk 1]
```diff
@@ -2,14 +2,17 @@
...
+    "fmt"
     "golang.org/x/sync/errgroup"
 	"github.com/kazhuravlev/options-gen/pkg/validator"
 	goplvalidator "github.com/go-playground/validator/v10"
 	{{- range $import := .imports }}
 	{{ $import -}}
 	{{- end }}
 )
+{{ if .hasValidation }}
+var v = goplvalidator.New()
```
Comment (kazhuravlev) [inline internal/generator/templates/options.go.tpl:14]: Default behaviour for options-gen is generating `options_generated.go` near the `options.go`. That means that we can have a name collision.

Consider changing the name to something like `_optionsValidator` or something little-bit more unique.

File: internal/generator/generator.go [hunk 2]
```diff
@@ -22,6 +22,26 @@ import (
 //go:embed templates/options.go.tpl
 var templates embed.FS
 
+var templateImports = []string{
```
Comment (kazhuravlev) [inline internal/generator/generator.go:25]: At first sight - that is unnecessary, and you can leave this right in the tpl file. options-gen will invoke the imports.Process for code formatting. But maybe I do not see something?

**Review by kazhuravlev — CHANGES_REQUESTED** [review]
Thank you for your contribution!

File: internal/generator/templates/options.go.tpl (hunk 1 above)
Comment (Antonboom) [reply internal/generator/templates/options.go.tpl:14]: nice point
1) Rename to `_validator461e464ebed9`.
2) Open https://github.com/kazhuravlev/options-gen/issues/15.

File: internal/generator/generator.go (hunk 2 above)
Comment (Antonboom) [reply internal/generator/generator.go]: Yes, I was surprised, but I received conflicts with "errors" and with "fmt" twice in different cases.
Currently I cannot reproduce it, lol.

Remove.

**Comment by kazhuravlev** [comment]
Perfect!

**Pull request merged**